backend/src/main/java/com/capstone/moneytree/service/impl/DefaultUserService.java

```diff
@@ -81,11 +81,23 @@ public User createUser(User user) {
 
       userDao.save(user);
 
-      LOG.info("Created user: {}", user.getFirstName());
+      LOG.info("Created user: {}", user.getUsername());
 
       return user;
    }
 
+   @Override
+   public User updateUser(User userToUpdate, User user) {
```

Comment:
this endpoint shouold not have two user object. only one. we get the id of that user, call the userDao to fetch the user from the db, if the user exist then we validate the newly sent user and then update otherwise we return an error if the user is not valid. we also return an error there is no such user in the database